bundles/core/org.openhab.core.jsr223/src/main/java/org/openhab/core/jsr223/internal/engine/scriptmanager/Script.java

```diff
@@ -68,52 +78,133 @@ public Script(ScriptManager scriptManager, File file) throws FileNotFoundExcepti
+						//therefore:
+						+"BusEvent 				= Java.type('org.openhab.model.script.actions.BusEvent'),\n"
+						+"be 					= Java.type('org.openhab.model.script.actions.BusEvent'),\n"
+						
+						+"transform 			= oh.getAction('Transformation').static.transform,\n"
+						
+						//Item
+						+"getItem 				= ItemRegistry.getItem,\n"
+						+"postUpdate 			= BusEvent.postUpdate,\n"
+						+"sendCommand 			= BusEvent.sendCommand,\n"
+						
+						//System
+						+"FileUtils 			= Java.type('org.apache.commons.io.FileUtils'),\n"
+						+"File 					= Java.type('java.io.File'),\n"
+						
+						+"ohEngine				= 'javascript';\n"
+						
+						//Helper Functions and Libs eventually later a lib Folder for default (Auto) loaded Libraries
+						//Bas64: https://gist.github.com/ncerminara/11257943
+						//+"load('configurations/scripts/jslib/b64.js');\n"
+						);
+			} catch (ScriptException e) {
+				logger.error("ScriptException in initializeSciptGlobals while importing default-classes: ", e);
+			}
+		}else{
+			engine.put("RuleSet", 				RuleSet.class);
+			engine.put("Rule", 					Rule.class);
+			engine.put("ChangedEventTrigger", 	ChangedEventTrigger.class);
+			engine.put("CommandEventTrigger", 	CommandEventTrigger.class);
+			engine.put("Event", 				Event.class);
+			engine.put("EventTrigger", 			EventTrigger.class);
+			engine.put("ShutdownTrigger", 		ShutdownTrigger.class);
+			engine.put("StartupTrigger", 		StartupTrigger.class);
+			engine.put("TimerTrigger", 			TimerTrigger.class);
+			engine.put("TriggerType", 			TriggerType.class);
+			engine.put("BusEvent", 				BusEvent.class);
+			engine.put("be", 					BusEvent.class);
+			engine.put("PersistenceExtensions", PersistenceExtensions.class);
+			engine.put("pe", 					PersistenceExtensions.class);
+			engine.put("oh", 					Openhab.class);
+			engine.put("State", 				State.class);
+			engine.put("Command", 				Command.class);
+			engine.put("ItemRegistry", 			scriptManager.getItemRegistry());
+			engine.put("ir", 					scriptManager.getItemRegistry());
+			engine.put("DateTime", 				DateTime.class);
+			engine.put("StringUtils", 			StringUtils.class);
+			engine.put("URLEncoder", 			URLEncoder.class);	
+			engine.put("FileUtils", 			FileUtils.class);	
+			engine.put("File", 					File.class);			
+	
+			// default types, TODO: auto import would be nice
+			engine.put("CallType", 				CallType.class);
+			engine.put("DateTimeType", 			DateTimeType.class);
+			engine.put("DecimalType", 			DecimalType.class);
+			engine.put("HSBType", 				HSBType.class);
+			engine.put("IncreaseDecreaseType", 	IncreaseDecreaseType.class);
+			engine.put("OnOffType", 			OnOffType.class);
+			engine.put("OpenClosedType", 		OpenClosedType.class);
+			engine.put("PercentType", 			PercentType.class);
+			engine.put("PointType", 			PointType.class);
+			engine.put("StopMoveType", 			StopMoveType.class);
+			engine.put("UpDownType", 			UpDownType.class);
+			engine.put("StringType", 			StringType.class);
+		}
```

Comment:
please extract into it's own initMethod for the sake of readability of this general init method